README.creole

```diff
@@ -47,7 +47,7 @@ For all functionality (and running the unittests) these modules are needed:
 Convert creole markup to html code:
 {{{
 >>> from creole import creole2html
->>> creole2html("This is **creole //markup//**")
+>>> creole2html(u"This is **creole //markup//**")
```

Comment:
Why add this? Makes only sense with Python v2

Reply:
It makes it consistent with the rest of the file's documentation with prefixes the string with `u`. I've confirmed that the example fails in Python 2 without it. Having the prefix does no harm in Python 3.

Even, I'm happy to take it back out. We might also want to remove it from the other code snippets as well. And then document in the text that the routines are expecting unicode strings.

Or, I could modify the routines themselves to convert non-unicode into unicode strings automatically. That might be a really good idea.

Just let me know.